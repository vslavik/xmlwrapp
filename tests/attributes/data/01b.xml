<root xmlns:test="http://pmade.org/namespace/test" a="a" test:a="ta" b="b" test:b="tb"/>
