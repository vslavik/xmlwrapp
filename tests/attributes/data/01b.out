a=a
a=ta
b=b
b=tb
